src/components/Commit.vue

```diff
@@ -157,6 +158,9 @@ export default {
     ...mapGetters([
       'currentNetwork',
     ]),
+    shouldShowInfoModal() {
+      return (!this.$store.state.acceptCommitInfo && !storage.get('commitInfoAccepted'));
```

Comment:
This likely does not need to directly check storage. It can be managed in mutations with an action to load from storage on mount.

Reply:
Also sorted in this [commit](https://github.com/aphtoken/aph-desktop-web-dex/pull/6/commits/125ff4b3bb1d2b9c7d482c2fc9fc96b7508b8909).